### ISSUE TITLE
Add CSS classes to R code highlighted using downlit

### ISSUE THREAD

**Issue by mattwarkentin** [rstudio/distill#283]
Hi,

Would you be willing to add the CSS classes `"sourceCode r"` to the `<pre>` and `<code>` tags inside the the surrounding `<div class="sourceCode">` for code chunks? This would make `R` code chunks more targetable by CSS selectors, and these are the selectors one gets when not using `downlit` for highlighting within `distill`.

The relevant code is:
https://github.com/rstudio/distill/blob/6dfc29d4bd771d3fe65be19a2de0ebc396a15fe3/R/distill_article.R#L389-L391

Which could be replaced with:
```r
x <- paste0("<div class=\"sourceCode\">",
                    "<pre class=\"sourceCode r\">",
                    "<code class=\"sourceCode r\">",
                    code,
                    "</code></pre></div>")
```

Currently, non-`R` code chunks have the expected classes (e.g. `python`):
```html
<div class="sourceCode">
  <pre class="sourceCode python">
    <code class="sourceCode python">
    "some python code..."
    </code>
  </pre>
</div>
```
While `R` code chunks highlighted using `downlit` look like:
```html
<div class="sourceCode">
  <pre>
    <code>
    "some R code..."
    </code>
  </pre>
</div>
```
And when `highlight_downlit: false`
```html
<div class="sourceCode">
  <pre class="sourceCode r">
    <code class="sourceCode r">
    "some R code..."
    </code>
  </pre>
</div>
```
Happy to submit a PR as I have tested this out on a fork.

**Comment by jjallaire** [comment]
Sure, a PR would be great!

**Comment by mattwarkentin** [comment]
Closed by #285
